_includes/navigation.html

```diff
@@ -37,6 +37,11 @@
                     <span class="label label-pill label-info">{{ site.data.future-episodes.size }}</span>
                 </a>
             </li>
+             <li class="nav-item">
+                <a class="nav-link" href="http://werocktech.com/awesome-tech-communities/" target="_blank">
+                    <i class="fa fa-handshake-o" aria-hidden="true"></i> Communities
```

Comment:
No worries, it happens to us all 😄 . Probably time to update though or switch to `cdn` so easier to update going forward 🤓